tests/ProjectMembershipRolesTest.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace Keboola\ManageApiTest;
+
+use Keboola\ManageApi\Client;
+use Keboola\ManageApi\ClientException;
+
+class ProjectMembershipRolesTest extends ClientMfaTestCase
+{
+    /** @var array */
+    private $organization;
+
+    /** @var array */
+    private $project;
+
+    private const ROLE_GUEST = 'guest';
+
+    /** @var Client */
+    private $guestRoleMemberClient;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => $this->normalUserWithMfa['email']]);
+
+        $this->organization = $this->normalUserWithMfaClient->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+
+        $this->project = $this->normalUserWithMfaClient->createProject($this->organization['id'], ['name' => ProjectMembershipRolesTest::class]);
+
+        $this->normalUserWithMfaClient->addUserToProject(
+            $this->project['id'],
+            [
+                'email' => $this->normalUser['email'],
+                'role' => self::ROLE_GUEST,
+            ]
+        );
+
+        $member = $this->findProjectUser($this->project['id'], $this->normalUser['email']);
+        $this->assertEquals(self::ROLE_GUEST, $member['role']);
```

Comment:
Tohle je imho na samostatný test. Dal bych to do samostaného testu. Zkrátí se tím setup a hlavně se to bude volat jenom jednou.

Reply:
Vyhodil jsem to. Uz je na to test `ProjectTests::testAddUserToProjectWithRole()`